arcgis-ios-sdk-samples/Scenes/Open mobile scene (scene package)/README.md

```diff
@@ -0,0 +1,38 @@
+# Open Mobile Scene Package
```

Comment:
Title in read me and sample name doesn't match

Reply:
Good catch! Thanks.